vp-docs/guide/advanced/README.md

```diff
@@ -83,6 +83,114 @@ Sometimes you might want to customize column headers. You can do that in the fol
 </vue-good-table>
 ```
 
+## Custom column filters
+Sometimes you might want a custom filter. You can do that in the following way:
+```html
+<vue-good-table
+  :columns="columns"
+  :rows="rows">
+  <template slot="column-filter" slot-scope="props">
+    <my-custom-filter
+      v-if="props.column.filterOptions.customFilter"       
+      @input="handleCustomFilter"/>
+  </template>
+</vue-good-table>
+```
+Add a custom property in your `columns` to conditionally render the `custom-filter` slot where needed. 
+```javascript
+columns: [
+  {
+    label: 'Name',
+    field: 'name'
+  },
+  {
+    label: 'Category',
+    field: 'category'
+  },
+  {
+    label: 'Statistics',
+    field: 'statistics',
+    filterOptions: {
+      customFilter: true
+    }
+  } 
+]
+// in your methods
+handleCustomFilter(value) {
+  // filtering logic here
+}
+```
+You can add a function to handle the filtering logic in your own component, or optionally `updateFilters` can be used. 
+The `updateFilters` is a method in `vue-good-table` and will include your custom filter value with the other column filters. 
+```html
+<vue-good-table
+  :columns="columns"
+  :rows="rows">
+  <template slot="column-filter" slot-scope="{ column, updateFilters }">
+    <my-custom-filter
+      v-if="column.customFilter"       
+      @input="(value) => updateFilters(column, value)"/>
+  </template>
+</vue-good-table>
+```
+In your columns, you may want to display the value from one property but need to filter on another. 
+If you set a `slotFilterField` in your filterOptions, that property will be used for the custom filter slot. 
+```javascript
+{
+  label: 'Name',
+  field: 'name.displayName',
+  filterOptions: {
+    customFilter: true,
+    slotFilterField: 'name.id'
+  }
+}
+```
+::: tip Upgrading from versions 2.19.0-2.20.0
```

Comment:
@xaksis I wasn't sure where it was desirable to have the migration guide. I notice there are some things like that in the wiki, but wasn't sure if I can/should contribute to that. 

I just put it here for now, but let me know if different.